docs/_rpc/ns-admin.md

```diff
@@ -90,6 +125,10 @@ overlay protocol, as well as specialized information added by each of the runnin
 }
 ```
 
+### admin_peerEvents
+
+PeerEvents creates an [RPC subscription](/docs/rpc/pubsub) which receives peer events from the node's p2p server.
```

Comment:
Perhaps would be nice to enumerate the possible events that the user might receive